src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -695,13 +695,13 @@ <h2 class="modal-title">
                 <i
                   ng-switch-when="encrypt"
                   class="glyphicon glyphicon-question-sign"
-                  uib-tooltip="Customised email acknowledgement to the respondent."
+                  uib-tooltip="Customize an email acknowledgement to respondents"
```

Comment:
British spelling?

Reply:
also if the messages are identical then we can remove the `ng-switch`

Reply:
To remove the switch statement (since the messages are identical), delete the `ng-switch` directive on L694 and the `ng-switch-when` directive on L696. Then delete the entire `<i>` element with the `ng-switch-default` directive.